packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -8,7 +8,7 @@
         <h3
             data-test-id="checkout-issue-modal-title"
             class="u-noSpacing">
-            {{ $t(`errorMessages.checkoutIssues.${errorCode}.title`) }}
+            {{ $t(`errorMessages.checkoutIssues.${errorCode}.title`, { serviceType }) }}
```

Comment:
If I'm reading the code correctly, this is passed in by `Checkout.vue` direct from the store, where it can be one of the `CHECKOUT_METHOD_COLLECTION` or `CHECKOUT_METHOD_DELIVERY` constants.

I think the Error dialog should be interpolating this via a translation string, which will be more important when we roll out to AU, where 'collection' is known as 'pick-up'.

Reply:
Very good point. I had actually thought of that then forgot during the implementation so I'll change that now! Thanks

Reply:
The change is also not reflected in the Changelog